flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBResourceContainer.java

```diff
@@ -59,6 +59,9 @@
 public final class RocksDBResourceContainer implements AutoCloseable {
     private static final Logger LOG = LoggerFactory.getLogger(RocksDBResourceContainer.class);
 
+    // the filename length limit is 255 on most operating systems
+    private static final int INSTANCE_PATH_LENGTH_LIMIT = 255 - "_LOG".length();
```

Comment:
Just a minor question.
why we need to reduce the length of "_LOG"?

Reply:
Because when log dir is set, rocksdb will add the suffix '_LOG' to the instance path as the log file name, you can check it in 
https://github.com/ververica/frocksdb/blob/FRocksDB-6.20.3/file/filename.cc#L31